### PR TITLE
Spelling Mistakes and Incorrect use of Grammar

### PR DESCRIPTION
**Pull request by ArmanChand**

### CODE REVIEW

File: source/configure/email-templates.rst [hunk 1]
```diff
@@ -3,14 +3,14 @@ Email Templates
 
 Mattermost has a few email templates that are sent out when a specific event occurs.
 Most of the time these templates do not need to be modified.
-In case additional modifications are necessary, all available props in each email are listed below.
+In case of additional modifications are necessary, all available props in each email are listed below.
```
Comment (haardikdharma10) [inline source/configure/email-templates.rst:6]: This change doesn't seem right to me. Please revert this.

Comment (ArmanChand) [reply source/configure/email-templates.rst]: removed. Thanks @haardikdharma10

**Review by cwarnermm — APPROVED** [review]
Thank you for this docs contribution! We really appreciate the spelling and typo corrections!

**Comment by ArmanChand** [comment]
@cwarnermm , kindly merge this PR

**Review by justinegeffen — APPROVED** [review]
Great changes, thank you!

**Pull request merged**